src/components/SimpleSlider.js

```diff
@@ -0,0 +1,170 @@
+const React = require('react');
+const ReactDOM = require('react-dom');
+const _noop = require('lodash/noop');
+const _merge = require('lodash/merge');
+
+const StyleConstants = require('../constants/Style');
+
+const SimpleSlider = React.createClass({
+  propTypes: {
+    disabled: React.PropTypes.bool,
+    onPercentChange: React.PropTypes.func.isRequired,
+    percent: React.PropTypes.number.isRequired,
+    selectedColor: React.PropTypes.string,
+    styles: React.PropTypes.object
+  },
+
+  getDefaultProps () {
+    return {
+      selectedColor: '#359BCF'
+    };
+  },
+
+  getInitialState () {
+    const disabled = this.props.disabled || false;
```

Comment:
It doesn't seem like this needs to be set on the state. You can just use `this.props.disabled` in your checks. If it isn't provided, it would return false.